website/api/goldparse.jade

```diff
@@ -185,7 +185,7 @@ p
         +cell The document that the BILUO tags refer to.
 
     +row
-        +cell #[code entities]
+        +cell #[code tags]
```

Comment:
This is more debateable. People often think of tags as part-of-speech tags though, and the description does say what the function expects.

Reply:
`tags` is the name of the argument that that function actually uses (see https://github.com/explosion/spaCy/blob/386cec1/spacy/gold.pyx#L550), and I think it's useful to distinguish between these "entities" (B/I/L/U-prefixed or O strings) and the 3-tuple "entities" expected by `biluo_tags_from_offsets`.